frontend/components/header/user-dropdown.tsx

```
@@ -1,12 +1,12 @@
 "use client";
-
 import { getInitials, condenseRoleTitles } from "@/lib/utils";
 import { AuthorizedUser, User } from "@/types";
 import {
   ChevronDownIcon,
   LogOutIcon,
   PersonStanding,
```

ESLint

[error] 'PersonStanding' is defined but never used.

(@typescript-eslint/no-unused-vars)